src/showcase/app/shared/component-viewer/example-viewer/example-viewer.component.ts

```diff
@@ -21,58 +21,72 @@ export class ExampleViewerComponent implements OnInit {
   @Input() exampleData: ExampleData;
   exampleCodes: Observable<ExampleCode[]>;
   showSource = false;
-  private _exampleFileOrder = ['html', 'ts', 'css'];
-  constructor(private _htmlLoader: HtmlLoader, private _route: ActivatedRoute) {}
-
-  // Returns the path to the html file for a given example file.
-  private static _convertToHtmlFilePath(filePath: string): string {
-    return filePath.replace(/(.*)[.](html|ts|css)/, '$1-$2.html');
-  }
-
-  // Get the example name from the selector name
-  private static _convertToExampleName(selectorName: string): string {
-    return selectorName.replace('sbb-', '').replace('-example', '');
-  }
+  private _defaultExtensionsOrder = ['html', 'ts', 'css'];
 
-  // Get the extension for a given file path
-  private static _getFileExtension(filePath: string): string {
-    return filePath.split('.').pop();
-  }
+  constructor(private _htmlLoader: HtmlLoader, private _route: ActivatedRoute) {}
 
   ngOnInit(): void {
     this.exampleCodes = combineLatest(
-      this.exampleData.exampleFiles.map((exampleFile) =>
-        this._createLoader(
-          ExampleViewerComponent._convertToExampleName(this.exampleData.selectorName),
-          exampleFile
-        ).pipe(
-          map((code) => ({
-            label: ExampleViewerComponent._getFileExtension(exampleFile),
-            code,
-          }))
+      this.exampleData.exampleFiles
+        .filter(
```

Comment:
Sorry, this was probably a misunderstanding from my wording.
We want to show all example files, but the "default" files should only have the extension as the label. The other example files should be displayed with the "full" file name.